pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4901,7 +4901,7 @@
       "InboxOn": {
         "netcoreapp2.0": "4.1.1.0",
         "netcoreapp2.1": "4.2.0.0",
-        "uap10.0.16300": "4.1.2.0"
+        "uap10.0.16300": "4.2.0.0"
```

Comment:
It's weird this isn't flagged as a conflict

Reply:
Unsure what you mean here. Do each of these entries need unique version numbers?

Reply:
No, I just meant that I added some changes on master on these lines as well so I would have expected that this change would have caused a conflict. The change itself is fine.